src/markdown-pages/install-with-kurl/upgrading.md

```diff
@@ -21,7 +21,10 @@ For example, it is possible to upgrade directly from Kubernetes 1.22 to 1.24, bu
 
 ## Container Runtimes
 
-Existing versions of docker and containerd will never be upgraded by the install script.
+If the install script detects an upgrade of a container runtime (`Docker` or `Containerd`) is required, then the new versions provided will be installed.
+For example, if you have a cluster using Containerd version `1.6.4` and then you modify the version in your spec to `1.6.18` and re-run the kURL script, the installer will upgrade Containerd to the newly specified version.
```

Comment:
This PR looks good to me and I approved already, but just wanted to offer one suggestion that lines 24 and 25 might not be necessary as they describe normal update behavior. (As in, it sounds like lines 24 and 25 describe the same process that happens when updating any type of add-on, not just Docker/Containerd.)

Reply:
I think it is good clarifies because the user might not know that container runtimes works as any addon
and it is possible to upgrade one by the other. (no harm to in describe) However, if you prefer to change please feel to make a suggestion and I will be happy to accept.

Reply:
Sounds good! Totally up to you if you think it is valuable to include this information. Just wanted to check :)